src/bonuses.cpp

```diff
@@ -58,6 +61,39 @@ affected_stat affected_stat_from_string( const std::string &s )
     return AFFECTED_NULL;
 }
 
+static const std::map<affected_stat, std::string> affected_stat_map_translation = {{
+        std::make_pair( AFFECTED_HIT, translate_marker( "Accuracy" ) ),
+        std::make_pair( AFFECTED_DODGE, translate_marker( "Dodge" ) ),
+        std::make_pair( AFFECTED_BLOCK, translate_marker( "Block" ) ),
+        std::make_pair( AFFECTED_SPEED, translate_marker( "Speed" ) ),
+        std::make_pair( AFFECTED_MOVE_COST, translate_marker( "Move cost" ) ),
+        std::make_pair( AFFECTED_DAMAGE, translate_marker( "damage" ) ),
```

Comment:
```suggestion
        std::make_pair( AFFECTED_DAMAGE, translate_marker( "Damage" ) ),
```

Reply:
Since damage will always be displayed after the actual damage type it looks weird if it's capitalized: cut Damage  bash Damage etc.